demo/src/html/index.html.template

```diff
@@ -97,7 +97,7 @@
     <ha-demo></ha-demo>
     <%= renderTemplate('_js_base') %>
 
-    <script type="module" src="<%= latestDemoJS %>"></script>
+    <script type="module" src="<%= latestDemoJS %>" crossorigin="use-credentials"></script>
```

Comment:
Let's not update the demo either

Reply:
Done :) Not sure why the demo shouldn't demo how to set up things correctly though? It passes crossorigin="use-credentials" for the manifest?